### DISCUSSION TITLE
Object is created, but I cant find it. Help!

### DISCUSSION THREAD

**Discussion by JustinNewport** [EvenAR/node-simconnect#78] [Q&A] [answered]
Hi @EvenAR, 

In this simple bit of code, a windsock is indeed being created. The Total Occurrences of the object in the Statistics Profiler increases by 1. And I have confirmed that the lat/lon/alt of the object is correct. Any idea why I am not seeing the object? 

```
import {
    open,
    Protocol,
    SimConnectDataType,
    SimConnectPeriod,
    SimConnectConstants,
    InitPosition
} from '../../dist/index.js';


open('My app', Protocol.KittyHawk)
    .then(function ({ recvOpen, handle }) {

        console.log('Connected to', recvOpen.applicationName);

        const initPos = new InitPosition();
        initPos.latitude = 0.5923607347913556;
        initPos.longitude = 2.066137895854694;
        initPos.altitude = 119.67113267541139;
        initPos.onGround = true;


        const a = handle.aICreateSimulatedObject('Windsock', initPos, 101)

        console.log(a)
    })
    .catch((err) => {
        console.log(err);
    });
```

**Comment by EvenAR** [comment]
Hi! SimConnect uses the same coordinate system as Google Maps (WGS84). When I Google "0.5923607347913556, 2.066137895854694" I end up in the Atlantic ocean. Perhaps your lat/long are from a different coordinate reference system?

**Reply by JustinNewport** [reply]
Oh wow, I pulled my coordinates by running the below when spawning into a Free Flight at KLAX. I'll double check what might be going wrong. Thank you for your response.

```
import { SystemEvents, MSFS_API } from "./msfs-api.js";
import { SimVars } from "./simvars/index.js";

const api = new MSFS_API();

api.connect({
  retries: Infinity,
  retryInterval: 5,
  onConnect: (nodeSimconnectHandle) => connect(nodeSimconnectHandle),
  onRetry: (_retries, interval) =>
    console.log(`Connection failed: retrying in ${interval} seconds.`),
});

async function connect(nodeSimconnectHandle) {
  const [longitude, latitude, altitude] = await Promise.all([
    api.get("PLANE_LONGITUDE"),
    api.get("PLANE_LATITUDE"),
    api.get("PLANE_ALTITUDE"),
  ]);

  console.log(`Plane's Location: Longitude=${longitude.PLANE_LONGITUDE}, Latitude=${latitude.PLANE_LATITUDE}, Altitude=${altitude.PLANE_ALTITUDE}`);

}
```

**Reply by EvenAR** [reply, marked as answer]
Ahh, [msfs-simconnect-api-wrapper](https://github.com/Pomax/msfs-simconnect-api-wrapper/blob/25dc380c121b5ea8127ae42674d8e397b030a4db/simvars/aircraft-misc-variables.js#L154) retrieves lat/lon in radians. Try multiplying them with `180/π` to convert them to degrees, and I think it should work.
